main/src/cgeo/geocaching/connector/ga/GeocachingAustraliaConnector.java

```diff
@@ -21,11 +21,6 @@ public String getNameAbbreviated() {
         return "GCAU";
     }
 
-    @Override
-    public boolean getHttps() {
```

Comment:
As part of general cleanup and since you touched this code anyway - shouldn't this method be renamed from `getHttps` to `isHttps`? Since it's returning trivial boolean value and that's the common convention to use `is...` for bool methods...

Reply:
Makes sense.
Will change it.